src/js/apps/patients/sidebar/bulk-edit-actions_app.js

```diff
@@ -147,6 +147,7 @@ export default App.extend({
     });
   },
   onSubmit() {
+    this.modal.disableSubmit();
```

Comment:
with this, the green save button was staying active while awaiting submission.. particularly noticeable on a patient flow page..and if someone hammered those, because it's parallel calls they could really stack up.  There might be more to do here, but this seems better than it was

Reply:
Nice, that works well 👍 

One thought I had while testing this in my local sandbox. Should the `Cancel` option be changed to something else like `Clear`, `Clear Changes`, etc. (this sits next to the `Save` button in the sidebar)? Or just hide it once the `Save` button is clicked by the user?

Because once the `Save` button is clicked and the request is sent to the backend (where db updates are made), I don't believe that request can be cancelled. At that point, clicking `Cancel` just closes the sidebar, but doesn't really cancel anything.

Reply:
Yeah maybe on the cancel thing.  Worth bringing up, but it'll be true everywhere.  The naming of the button is a UX decision.